array/delete_nth.py

```diff
@@ -0,0 +1,42 @@
+"""
+Given a list lst and a number N, create a new list that contains each number of the list at most N times without reordering. 
+
+For example if N = 2, and the input is [1,2,3,1,2,1,2,3], you take [1,2,3,1,2], 
+drop the next [1,2] since this would lead to 1 and 2 being in the result 3 times, and then take 3, 
+which leads to [1,2,3,1,2,3]
+"""
+
+# Time complexity O(n)
+def delete_nth(order,max_e):
```

Comment:
I think Time Complexity for this is O(n<sup>2</sup>) since ans.count also isO(n).

Reply:
Yes, you're right. I meant to say O(n<sup>2</sup>) but somehow forgot to include that. That's what happens when you're programming at 3 am. 🗡